spec/factories/requests.rb

```diff
@@ -28,29 +28,48 @@ def random_request_items
     request_items { random_request_items }
     comments { "Urgent" }
     partner_user { ::User.partner_users.first || create(:partners_user) }
-  end
 
-  trait :started do
-    status { 'started' }
-  end
+    # For compatibility we can take in a list of request_items and turn it into a
+    # list of item_requests
+    trait :with_item_requests do
+      after(:build) do |request|
```

Comment:
I ended up making this an explicit trait so that it doesn't interfere with existing specs that don't pass in valid data.